etna/datasets/internal_datasets.py

```diff
@@ -126,6 +125,16 @@ def load_dataset(
             parse_dates=[0],
         )
         ts = TSDataset(data, freq=freq)
+        if "m3" in name:
```

Comment:
It isn't a good solution. We should think of smth more general that can be used in the future by other datasets' methods.